src-java/testing/test-library/src/main/java/org/openkilda/testing/service/tsdb/model/StatsResult.java

```diff
@@ -62,7 +62,7 @@ public boolean hasNonZeroValues() {
 
     public boolean hasNonZeroValuesAfter(long timestamp) {
         return dataPoints.keySet().stream()
-                .anyMatch(tstamp -> tstamp >= timestamp && dataPoints.get(tstamp) > 0);
+                .anyMatch(tstamp -> tstamp >= timestamp && dataPoints.get(tstamp) != 0);
```

Comment:
Is this fix for the TC 'Unable to ping one of the HA-subflows via periodic pings if related ISL is broken' or for some other HA-Flow-related cases?

Reply:
Yep.